js/tests/visual/toast.html

```diff
@@ -28,7 +28,7 @@ <h1>Toast <small>Bootstrap Visual Test</small></h1>
     <div class="notifications">
       <div id="toastAutoHide" class="toast" role="alert" aria-live="assertive" aria-atomic="true" data-delay="2000">
         <div class="toast-header">
-          <img class="rounded mr-2" data-src="holder.js/20x20?size=1&text=.&bg=#007aff" alt="">
+          <span class="d-block bg-primary rounded mr-2" style="width: 20px; height: 20px;"></span>
```

Comment:
This is just for internal usage hence why I just used a `span`. Alternatively, I could just inline the SVG hardcoded for this case. Maybe that'll be better since it'll be consistent with the rest.

Reply:
For a visual test this is fine.